src/components/OnboardingComponents/TourComponents/TourSteps.tsx

```diff
@@ -69,14 +71,14 @@ export default function TourSteps(): JSX.Element {
    * sets the simulation start date, and starts the tour once preferences are saved.
    */
   useEffect(() => {
-    if (activeTour && !showPopover && !showWelcomeDialog && !run) {
+    if (activeTour && !showPopover && !showWelcomeDialog && !run && scenarioList) {
       if (!savedUserDataSelection.current) {
         savedUserDataSelection.current = dataSelection; // Save the current data selection of the user so we can override it with different values during the tour
 
-        const maxDate = dataSelection.maxDate || '2024-07-08'; // Get the maximum date from the data selection
+        const maxDate = dataSelection.maxDate ?? '2024-07-08'; // Get the maximum date from the data selection
         dispatch(selectDate(maxDate)); // Set the simulation start date (purple line) to the maximum date
```

Comment:
@NXXR figure out why the specific date.

```suggestion
        const maxDate = dataSelection.maxDate ?? '2024-07-08'; // Get the maximum date from the data selection
        dispatch(selectDate(maxDate)); // Set the selected date (purple line) to the maximum date
```